src/base/StrBuf.cpp

```diff
@@ -0,0 +1,49 @@
+#include "StrBuf.hpp"
+#include <memory.h>
+
+//--------------------------------------------------------------------------------
```

Comment:
Can you please remove this comment?
Our coding convention does not require this kind of separator.